homeassistant/components/shelly/const.py

```diff
@@ -32,3 +32,38 @@
     "SL": "single_long",
     "LS": "long_single",
 }
+
+EVENT_SHELLY_CLICK = "shelly.click"
+
+ATTR_CLICK_TYPE = "click_type"
+ATTR_CHANNEL = "channel"
+ATTR_DEVICE = "device"
+CONF_SUBTYPE = "subtype"
+
+BASIC_TRIGGER_TYPES = {
+    "single_click",
+    "long_click",
+}
+
+SHBTN_1_TRIGGER_TYPES = {
+    "single_click",
+    "double_click",
+    "triple_click",
+    "long_click",
+}
+
+SUPPORTED_TRIGGER_TYPES = SHIX3_1_TRIGGER_TYPES = {
```

Comment:
Why are we not using same values as `INPUTS_EVENTS_DICT` ?

Reply:
I will use the same values (reached a conclusion that this would be the most clean solution and not adding another event type)